nltk/embedding/word2vec.py

```diff
@@ -0,0 +1,134 @@
+# Natural Language Toolkit: Word2Vec Wrapper through Gensim
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Long Duong <longdt219@gmail.com>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+Word2Vec Wrapping through Gensim. Allowing 
+- Training new model
+- Load the pre-trained model from nltk.corpus 
+- Run some demo tasks  
+"""
+from __future__ import absolute_import, division, unicode_literals
+import gensim
+
+
+
+class Word2Vec():
```

Comment:
```python
class Word2Vec(object):
```

I'm actually not sure whether a wrapper brings any value in comparison of using the wordsim directly.

It would be beneficial to wrap the gensim implementation to make it compatible with  some API. However, I find the gensim interface for accessing vectors nicer. 

Catching exceptions and returning `-1` or `None` instead may lead to strange bugs.

`get_vector()` should be `__getitem__()`, so it would be possible to access a word vector using `model[word]`.